scripts/grab_reference_docs.sh

```diff
@@ -48,7 +48,6 @@ COMPONENTS=(
     https://github.com/istio/istio.git@"${SOURCE_BRANCH_NAME}"@istioctl/cmd/istioctl@istioctl
     https://github.com/istio/istio.git@"${SOURCE_BRANCH_NAME}"@pilot/cmd/pilot-agent@pilot-agent
     https://github.com/istio/istio.git@"${SOURCE_BRANCH_NAME}"@pilot/cmd/pilot-discovery@pilot-discovery
-    https://github.com/istio/istio.git@"${SOURCE_BRANCH_NAME}"@security/cmd/istio_ca@istio_ca
```

Comment:
Is there doc moved, or need to be added, somewhere else that documents how to set some of the CA options? For example Istiod cmd options for CA?

Reply:
Not certain, I will defer to @howardjohn since the deletion was his change.

Reply:
The whole binary is gone. Any settings that can be used to configure Istiod CA should automagically show up in the pilot-discovery page